ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/RejectReason.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.divorce.ccd.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.ccd.sdk.types.CCD;
+import uk.gov.hmcts.ccd.sdk.types.ComplexType;
+import uk.gov.hmcts.reform.divorce.ccd.model.enums.RejectReasonList;
+
+import static uk.gov.hmcts.ccd.sdk.types.FieldType.FixedList;
+import static uk.gov.hmcts.ccd.sdk.types.FieldType.TextArea;
+
+@Data
+@Builder
+@AllArgsConstructor
+@ComplexType(generate = true)
```

Comment:
You shouldn't need to use the complex type annotation here, if it's reachable from your case data class then it should be generated.

I was planning on getting rid of the @ComplexType annotation and just having @CCD